config/locales/ca.yml

```diff
@@ -1,3319 +1,6 @@
...Totes les categoriesSense categories%{category_count} categoriesNom del tipus d'ordreCategories en les quals s'empraGuardant ordreL'ordre ha estat guardada correctamentHa ocorregut un error mentre s'estava guardant la comanda. Si us plau, actualitzi la pàgina i torni a intentar-ho.Productes en vendaProductes en lloguerOferta de serveisProductes regalatsSol·licitantPublicar anunciPersonalitzatCrear tipus d'ordreCrearCancelarNou tipus d'ordre '%{shape}' creatNo es pot crear un nou tipus d'ordre. Erro: %{error_msg}Editar el tipus d'ordre '%{shape}'DesarCancel·larS'han guardat els canvis al tipus d'ordre '%{shape}'No s'han pogut guardar els canvis. Error:%{error_msg}Borrar tipus d'ordre{"one"=>"Hi ha un registre de %{count} amb aquest tipus de comanda. Si suprimiu el tipus de comanda, aquest llistat es tancarà. Esteu segur que voleu suprimir el tipus de comanda?", "other"=>"Hi ha %{count} llistats amb aquest tipus de comanda. Si suprimiu el tipus de comanda, aquests llistats es tancaran. Esteu segur que voleu suprimir el tipus de comanda?"}No pots eliminar aquest tipus d'ordre perquè és l'únic tipus d'ordre que existeix a la teva comunitat.No pots eliminar aquest tipus d'ordre perquè en les següents categories només s'empra aquest tipus d'ordre: %{categories}Nomex: VendreEtiqueta el botó de checkoutex: Comprar{"one"=>"Hi ha %{count} anunci obert amb aquest tipus d'ordre. Si canvies qualsevol dels ajustaments de baix, aquest anunci no s'actualitzarà i mantindrà els ajustaments anteriors. Pots actualitzar els paràmetres de l'anunci editant-lo manualment. En cas que no vulguis que aquest anunci amb la configuració desactualitzats sigui visible en la teva comunitat, pots tancar-lo (desactivar-lo) fent clic al botó de sota.", "other"=>"Hi ha %{count}  anuncis oberts amb aquest tipus d'ordre. Si canvies qualsevol dels ajustaments de baix, aquests anuncis no s'actualitzaran i mantindran els ajustaments anteriors. Pots actualitzar la configuració dels anuncis editant -los manualment. En cas que no vulguis que aquests anuncis amb la configuració desactualitzats siguin visibles a la teva comunitat, pots tancar-los (desactivar-los) fent clic al botó de sota."}{"one"=>"Tanca l'anunci de %{count}", "other"=>"Tanca els anuncis de %{count}"}{"one"=>"Esteu segur que voleu tancar l'anunci %{count}?", "other"=>"Esteu segur que voleu tancar els anuncis de %{count}?"}Anuncis tancats correctamentTipus d'ordre '%{order_type}' eliminat correctamentPreu & pagamentPermetre als usuaris acceptar pagaments en líniaPermetre als venedors definir una tarifa d'enviamentPermetre als usuaris afegir preu als seus anuncisUnitats de preuSi habilita les unitats de preu, el preu de l'anunci es mostrarà com "preu per unitat de preu". Per exemple: "$39 per hora".Per peçaPer horaPer diaPer nitPer setmanaPer mesPer unitatNo es pot trobar el tipus d'ordre amb id: %{id}+ Añadir una unidad de precio personalizada...esborrarNova unitat de preuEtiquetaEtiqueta del selectorex: "Kg", "30 minuts",
```

Comment:
The contents of this file are somehow messed up, so this results in app not able to parse it.